amundsen_application/static/js/ducks/issue/sagas.ts

```diff
@@ -27,8 +27,20 @@ export function* getIssuesWatcher(): SagaIterator {
 export function* createIssueWorker(action: CreateIssueRequest): SagaIterator {
   try { 
     let response;
-    response = yield call(API.createIssue, action.payload.data);
+    const { key, title, description, owners, resourceName, resourcePath, sender} = action.payload;
+    response = yield call(API.createIssue, key, title, description);
     yield put((createIssueSuccess(response)));
+
+    // Send a notification to table owners if they exist 
```

Comment:
Curious what people think about this as a way to send an email- I felt like it made the most sense to trigger the event here. Another question I have is whether other people think the notification for sending an email should be hidden? Or maybe there should be some text on the modal that the table owners will be notified?